nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestUnpackContent.java

```diff
@@ -199,11 +199,12 @@ public void testFlowFileStreamV3() throws IOException {
         final TestRunner runner = TestRunners.newTestRunner(new UnpackContent());
         runner.setProperty(UnpackContent.PACKAGING_FORMAT, UnpackContent.PackageFormat.FLOWFILE_STREAM_FORMAT_V3.toString());
         runner.enqueue(dataPath.resolve("data.flowfilev3"));
+        runner.enqueue(dataPath.resolve("data.flowfilev3"));
 
-        runner.run();
+        runner.run(2);
 
-        runner.assertTransferCount(UnpackContent.REL_SUCCESS, 2);
-        runner.assertTransferCount(UnpackContent.REL_ORIGINAL, 1);
+        runner.assertTransferCount(UnpackContent.REL_SUCCESS, 4);
+        runner.assertTransferCount(UnpackContent.REL_ORIGINAL, 2);
```

Comment:
I would like to see this pattern (send 2 flowfiles) in other unit tests, too.  Can we add this to testTar(), testTarWithFilter(), testZip(), testZipWithFilter(), and testFlowFileStreamV2()?